601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/BookRepository.java

```diff
@@ -3,10 +3,11 @@
 import io.quarkus.qe.spring.data.model.Book;
 import java.util.List;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.Repository;
 import org.springframework.data.repository.query.Param;
 
-public interface BookRepository extends Repository<Book, Integer> {
+public interface BookRepository extends Repository<Book, Integer>, CrudRepository<Book, Integer> {
```

Comment:
This seems redundant, `CrudRepository` already extends `Repository`.
```suggestion
public interface BookRepository extends CrudRepository<Book, Integer> {
```